cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/release_plan_admission-segment-bridge.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  name: rhtap-segment-bridge-rpa
+  namespace: rh-managed-rhtap-ser-tenant
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+spec:
+  application: segment-bridge
+  displayName: ''
+  origin: rhtap-o11y-tenant
+  releaseStrategy: rhtap-service-push-strategy
```

Comment:
This will use Pyxis stage and `rh-policy`. You ok with that I guess.

Reply:
yeah, as  long as its not going to give me any hard time.

Reply:
this is fine